parser/prism/Translator.h

```diff
@@ -29,15 +32,17 @@ class Translator final {
     Translator &operator=(Translator &&) = delete;      // Move assignment
     Translator &operator=(const Translator &) = delete; // Copy assignment
 public:
-    Translator(Parser parser, core::GlobalState &gs) : parser(parser), gs(gs) {}
+    Translator(Parser parser, core::GlobalState &gs, core::FileRef file)
+        : parser(std::move(parser)), gs(gs), file(file) {}
 
     // Translates the given AST from Prism's node types into the equivalent AST in Sorbet's legacy parser node types.
     std::unique_ptr<parser::Node> translate(pm_node_t *node);
     std::unique_ptr<parser::Node> translate(const Node &node);
 
 private:
-    Translator(Parser parser, core::GlobalState &gs, bool isInMethodDef)
-        : parser(parser), gs(gs), isInMethodDef(isInMethodDef) {}
+    Translator(Parser parser, core::GlobalState &gs, core::FileRef file, bool isInMethodDef)
+        : parser(parser), gs(gs), file(file), isInMethodDef(isInMethodDef) {}
+    void reportError(core::LocOffsets loc, const std::string &message);
```

Comment:
Late to the party, but in a future PR, could you please move this method down to better match its order in the CC file?